src/ast/mod.rs

```diff
@@ -824,6 +846,36 @@ impl fmt::Display for Statement {
                 }
                 Ok(())
             }
+            Statement::Deallocate { name, all, prepare } => {
+                f.write_str("DEALLOCATE ")?;
+                if *prepare {
+                    f.write_str("PREPARE ")?;
+                }
```

Comment:
I see we're inconsistent about that, but I believe using the same style as in the Statement::CreateTable branch will yield smaller code (especially we figure out the ALL vs object name question above).